package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@fluvio/client",
-    "version": "5.0.0-alpha.1",
+    "version": "5.0.0-alpha.7",
```

Comment:
seems to be different version from others?

Reply:
yeah, had some debugging; had to bump the version to make sure worked.